---------------------------------------
Begin PBS Prologue Sat Oct  8 02:13:23 EDT 2022
Job ID:     5020801.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   imitation
Queue:      inferno
End PBS Prologue Sat Oct  8 02:13:23 EDT 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/utils/data/dataloader.py:478: UserWarning: This DataLoader will create 12 worker processes in total. Our suggested max number of worker in current system is 4, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name | Type           | Params
----------------------------------------
0 | net  | CNNAutoEncoder | 255 K 
----------------------------------------
255 K     Trainable params
0         Non-trainable params
255 K     Total params
1.023     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2022-10-08/AUTOENCODER exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  carnet_autoencoder_training[0m
=>> PBS: job killed: walltime 30009 exceeded limit 30000
---------------------------------------
Begin PBS Epilogue Sat Oct  8 10:33:32 EDT 2022
Job ID:     5020801.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   imitation
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=33:11:16,vmem=309058648kb,walltime=08:20:09,mem=4042004kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-006-35.pace.gatech.edu
End PBS Epilogue Sat Oct  8 10:33:32 EDT 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Sun Oct  9 02:51:13 EDT 2022
Job ID:     5024915.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   imitation
Queue:      inferno
End PBS Prologue Sun Oct  9 02:51:13 EDT 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/utils/data/dataloader.py:478: UserWarning: This DataLoader will create 12 worker processes in total. Our suggested max number of worker in current system is 4, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name | Type              | Params
-------------------------------------------
0 | net  | ResNetAutoencoder | 519 K 
-------------------------------------------
519 K     Trainable params
0         Non-trainable params
519 K     Total params
2.078     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2022-10-09/AUTOENCODER exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  carnet_autoencoder_training[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/nn/functional.py:3631: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  warnings.warn(
=>> PBS: job killed: walltime 30045 exceeded limit 30000
---------------------------------------
Begin PBS Epilogue Sun Oct  9 11:11:57 EDT 2022
Job ID:     5024915.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   imitation
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=33:13:59,vmem=320566620kb,walltime=08:20:45,mem=4163020kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-005-30.pace.gatech.edu
End PBS Epilogue Sun Oct  9 11:11:57 EDT 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Wed Oct 12 19:12:57 EDT 2022
Job ID:     5039725.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   imitation
Queue:      inferno
End PBS Prologue Wed Oct 12 19:12:57 EDT 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
[88mSkipping the block   |  carnet_autoencoder_training[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/utils/data/dataloader.py:478: UserWarning: This DataLoader will create 12 worker processes in total. Our suggested max number of worker in current system is 4, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name | Type              | Params
-------------------------------------------
0 | net  | ResNetAutoencoder | 525 K 
-------------------------------------------
525 K     Trainable params
0         Non-trainable params
525 K     Total params
2.102     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2022-10-12/SEGMENTATION exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  carnet_semseg_training[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/nn/functional.py:3631: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  warnings.warn(
=>> PBS: job killed: walltime 30016 exceeded limit 30000
---------------------------------------
Begin PBS Epilogue Thu Oct 13 03:33:13 EDT 2022
Job ID:     5039725.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   imitation
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=33:05:06,vmem=321528412kb,walltime=08:20:16,mem=4250176kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-01-004-32.pace.gatech.edu
End PBS Epilogue Thu Oct 13 03:33:13 EDT 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Sat Oct 15 02:49:02 EDT 2022
Job ID:     5051615.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   imitation
Queue:      inferno
End PBS Prologue Sat Oct 15 02:49:02 EDT 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
[88mSkipping the block   |  carnet_autoencoder_training[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/utils/data/dataloader.py:478: UserWarning: This DataLoader will create 12 worker processes in total. Our suggested max number of worker in current system is 4, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name | Type              | Params
-------------------------------------------
0 | net  | ResNetAutoencoder | 525 K 
-------------------------------------------
525 K     Trainable params
0         Non-trainable params
525 K     Total params
2.102     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2022-10-15/SEGMENTATION exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  carnet_semseg_training[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/nn/functional.py:3631: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  warnings.warn(
=>> PBS: job killed: walltime 30001 exceeded limit 30000
---------------------------------------
Begin PBS Epilogue Sat Oct 15 11:09:03 EDT 2022
Job ID:     5051615.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   imitation
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=33:16:30,vmem=322036812kb,walltime=08:20:01,mem=3649260kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-005-27.pace.gatech.edu
End PBS Epilogue Sat Oct 15 11:09:03 EDT 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Sun Oct 16 04:30:32 EDT 2022
Job ID:     5053478.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_segmentation
Queue:      inferno
End PBS Prologue Sun Oct 16 04:30:32 EDT 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
[88mSkipping the block   |  carnet_autoencoder_training[0m
[88mSkipping the block   |  carnet_semseg_training[0m
[88mSkipping the block   |  verify_autoencoder[0m
[88mSkipping the block   |  verify_segmentation[0m
[88mSkipping the block   |  carnet_training[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/nn/modules/lazy.py:178: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/utils/data/dataloader.py:478: UserWarning: This DataLoader will create 12 worker processes in total. Our suggested max number of worker in current system is 4, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/utilities/model_summary.py:438: UserWarning: A layer with UninitializedParameter was found. Thus, the total number of parameters detected may be inaccurate.
  warning_cache.warn(

  | Name | Type      | Params
-----------------------------------
0 | net  | ResCARNet | 562 K 
-----------------------------------
562 K     Trainable params
0         Non-trainable params
562 K     Total params
2.251     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2022-10-16/RNN_SEGMENTATION exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  rescarnet_training[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/nn/functional.py:3631: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  warnings.warn(
=>> PBS: job killed: walltime 30012 exceeded limit 30000
---------------------------------------
Begin PBS Epilogue Sun Oct 16 12:50:44 EDT 2022
Job ID:     5053478.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_segmentation
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=33:16:08,vmem=369369968kb,walltime=08:20:12,mem=5015752kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-005-28.pace.gatech.edu
End PBS Epilogue Sun Oct 16 12:50:44 EDT 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Sun Oct 23 02:25:42 EDT 2022
Job ID:     5083143.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_segmentation
Queue:      inferno
End PBS Prologue Sun Oct 23 02:25:42 EDT 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name | Type           | Params
----------------------------------------
0 | net  | CNNAutoEncoder | 296 K 
----------------------------------------
296 K     Trainable params
0         Non-trainable params
296 K     Total params
1.187     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2022-10-23/AUTOENCODER exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  carnet_autoencoder_training[0m
Validation sanity check: 0it [00:00, ?it/s]Traceback (most recent call last):
  File "/storage/scratch1/4/hmanjunatha6/carnet/train.py", line 100, in <module>
    trainer.fit(model)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 740, in fit
    self._call_and_handle_interrupt(
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 685, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 777, in _fit_impl
    self._run(model, ckpt_path=ckpt_path)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1199, in _run
    self._dispatch()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1279, in _dispatch
    self.training_type_plugin.start_training(self)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/plugins/training_type/training_type_plugin.py", line 202, in start_training
    self._results = trainer.run_stage()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1289, in run_stage
    return self._run_train()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1311, in _run_train
    self._run_sanity_check(self.lightning_module)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1371, in _run_sanity_check
    self._evaluation_loop._reload_evaluation_dataloaders()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/loops/dataloader/evaluation_loop.py", line 170, in _reload_evaluation_dataloaders
    self.trainer.reset_val_dataloader()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/data_loading.py", line 547, in reset_val_dataloader
    self.num_val_batches, self.val_dataloaders = self._reset_eval_dataloader(
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/data_loading.py", line 452, in _reset_eval_dataloader
    dataloaders = self.request_dataloader(mode, model=model)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/data_loading.py", line 607, in request_dataloader
    dataloader = source.dataloader()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/connectors/data_connector.py", line 296, in dataloader
    return self.instance.trainer.call_hook(self.name, pl_module=self.instance)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1501, in call_hook
    output = model_fx(*args, **kwargs)
  File "/storage/scratch1/4/hmanjunatha6/carnet/src/models/encoding.py", line 58, in val_dataloader
    return self.data_loader['validation']
KeyError: 'validation'
Sun Oct 23 02:29:01 EDT 2022
Successfully completed
---------------------------------------
Begin PBS Epilogue Sun Oct 23 02:29:01 EDT 2022
Job ID:     5083143.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_segmentation
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=00:00:09,vmem=10443104kb,walltime=00:03:22,mem=1578592kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-01-004-33.pace.gatech.edu
End PBS Epilogue Sun Oct 23 02:29:01 EDT 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Sun Oct 23 12:50:56 EDT 2022
Job ID:     5083373.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_segmentation
Queue:      inferno
End PBS Prologue Sun Oct 23 12:50:56 EDT 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/utils/data/dataloader.py:478: UserWarning: This DataLoader will create 12 worker processes in total. Our suggested max number of worker in current system is 4, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name | Type           | Params
----------------------------------------
0 | net  | CNNAutoEncoder | 296 K 
----------------------------------------
296 K     Trainable params
0         Non-trainable params
296 K     Total params
1.187     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2022-10-23/AUTOENCODER exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  carnet_autoencoder_training[0m
=>> PBS: job killed: walltime 30043 exceeded limit 30000
---------------------------------------
Begin PBS Epilogue Sun Oct 23 21:11:38 EDT 2022
Job ID:     5083373.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_segmentation
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=33:15:38,vmem=314829688kb,walltime=08:20:43,mem=4317324kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-005-28.pace.gatech.edu
End PBS Epilogue Sun Oct 23 21:11:38 EDT 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Sun Oct 23 22:28:52 EDT 2022
Job ID:     5085056.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_segmentation
Queue:      inferno
End PBS Prologue Sun Oct 23 22:28:52 EDT 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/utils/data/dataloader.py:478: UserWarning: This DataLoader will create 12 worker processes in total. Our suggested max number of worker in current system is 4, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name | Type           | Params
----------------------------------------
0 | net  | CNNAutoEncoder | 296 K 
----------------------------------------
296 K     Trainable params
0         Non-trainable params
296 K     Total params
1.187     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2022-10-23/AUTOENCODER exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  carnet_autoencoder_training[0m
=>> PBS: job killed: walltime 30051 exceeded limit 30000
---------------------------------------
Begin PBS Epilogue Mon Oct 24 06:49:42 EDT 2022
Job ID:     5085056.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_segmentation
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=33:01:36,vmem=321478764kb,walltime=08:20:51,mem=4362216kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-006-33.pace.gatech.edu
End PBS Epilogue Mon Oct 24 06:49:42 EDT 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Tue Oct 25 00:10:11 EDT 2022
Job ID:     5092142.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_segmentation
Queue:      inferno
End PBS Prologue Tue Oct 25 00:10:11 EDT 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name | Type           | Params
----------------------------------------
0 | net  | CNNAutoEncoder | 296 K 
----------------------------------------
296 K     Trainable params
0         Non-trainable params
296 K     Total params
1.187     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2022-10-25/AUTOENCODER exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  carnet_autoencoder_training[0m
Epoch    19: reducing learning rate of group 0 to 9.5000e-04.
Epoch    25: reducing learning rate of group 0 to 9.0250e-04.
=>> PBS: job killed: walltime 30037 exceeded limit 30000
---------------------------------------
Begin PBS Epilogue Tue Oct 25 08:30:57 EDT 2022
Job ID:     5092142.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_segmentation
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=32:31:05,vmem=107969168kb,walltime=08:20:47,mem=3175940kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-005-28.pace.gatech.edu
End PBS Epilogue Tue Oct 25 08:30:57 EDT 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Tue Oct 25 15:51:52 EDT 2022
Job ID:     5094425.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_segmentation
Queue:      inferno
End PBS Prologue Tue Oct 25 15:51:52 EDT 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
[88mSkipping the block   |  carnet_autoencoder_training[0m
[88mSkipping the block   |  carnet_semseg_training[0m
[88mSkipping the block   |  verify_autoencoder[0m
[88mSkipping the block   |  verify_segmentation[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/nn/modules/lazy.py:178: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/utilities/model_summary.py:438: UserWarning: A layer with UninitializedParameter was found. Thus, the total number of parameters detected may be inaccurate.
  warning_cache.warn(

  | Name | Type   | Params
--------------------------------
0 | net  | CARNet | 395 K 
--------------------------------
395 K     Trainable params
0         Non-trainable params
395 K     Total params
1.583     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2022-10-25/CARNET exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  carnet_training[0m
Traceback (most recent call last):
  File "/storage/scratch1/4/hmanjunatha6/carnet/train.py", line 272, in <module>
    trainer.fit(model)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 740, in fit
    self._call_and_handle_interrupt(
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 685, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 777, in _fit_impl
    self._run(model, ckpt_path=ckpt_path)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1199, in _run
    self._dispatch()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1279, in _dispatch
    self.training_type_plugin.start_training(self)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/plugins/training_type/training_type_plugin.py", line 202, in start_training
    self._results = trainer.run_stage()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1289, in run_stage
    return self._run_train()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1311, in _run_train
    self._run_sanity_check(self.lightning_module)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1371, in _run_sanity_check
    self._evaluation_loop._reload_evaluation_dataloaders()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/loops/dataloader/evaluation_loop.py", line 170, in _reload_evaluation_dataloaders
    self.trainer.reset_val_dataloader()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/data_loading.py", line 547, in reset_val_dataloader
    self.num_val_batches, self.val_dataloaders = self._reset_eval_dataloader(
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/data_loading.py", line 452, in _reset_eval_dataloader
    dataloaders = self.request_dataloader(mode, model=model)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/data_loading.py", line 607, in request_dataloader
    dataloader = source.dataloader()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/connectors/data_connector.py", line 296, in dataloader
    return self.instance.trainer.call_hook(self.name, pl_module=self.instance)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1501, in call_hook
    output = model_fx(*args, **kwargs)
  File "/storage/scratch1/4/hmanjunatha6/carnet/src/models/encoding.py", line 61, in val_dataloader
    return self.data_loader['validation']
KeyError: 'validation'
Tue Oct 25 15:52:08 EDT 2022
Successfully completed
---------------------------------------
Begin PBS Epilogue Tue Oct 25 15:52:08 EDT 2022
Job ID:     5094425.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_segmentation
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=00:00:10,vmem=10780892kb,walltime=00:00:17,mem=1592448kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-005-28.pace.gatech.edu
End PBS Epilogue Tue Oct 25 15:52:08 EDT 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Tue Oct 25 15:55:43 EDT 2022
Job ID:     5094433.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_segmentation
Queue:      inferno
End PBS Prologue Tue Oct 25 15:55:43 EDT 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
[88mSkipping the block   |  carnet_autoencoder_training[0m
[88mSkipping the block   |  carnet_semseg_training[0m
[88mSkipping the block   |  verify_autoencoder[0m
[88mSkipping the block   |  verify_segmentation[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/nn/modules/lazy.py:178: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/utilities/model_summary.py:438: UserWarning: A layer with UninitializedParameter was found. Thus, the total number of parameters detected may be inaccurate.
  warning_cache.warn(

  | Name | Type   | Params
--------------------------------
0 | net  | CARNet | 395 K 
--------------------------------
395 K     Trainable params
0         Non-trainable params
395 K     Total params
1.583     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2022-10-25/CARNET exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  carnet_training[0m
Traceback (most recent call last):
  File "/storage/scratch1/4/hmanjunatha6/carnet/train.py", line 272, in <module>
    trainer.fit(model)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 740, in fit
    self._call_and_handle_interrupt(
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 685, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 777, in _fit_impl
    self._run(model, ckpt_path=ckpt_path)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1199, in _run
    self._dispatch()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1279, in _dispatch
    self.training_type_plugin.start_training(self)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/plugins/training_type/training_type_plugin.py", line 202, in start_training
    self._results = trainer.run_stage()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1289, in run_stage
    return self._run_train()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1319, in _run_train
    self.fit_loop.run()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/loops/base.py", line 145, in run
    self.advance(*args, **kwargs)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/loops/fit_loop.py", line 234, in advance
    self.epoch_loop.run(data_fetcher)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/loops/base.py", line 145, in run
    self.advance(*args, **kwargs)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/loops/epoch/training_epoch_loop.py", line 193, in advance
    batch_output = self.batch_loop.run(batch, batch_idx)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/loops/base.py", line 145, in run
    self.advance(*args, **kwargs)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/loops/batch/training_batch_loop.py", line 88, in advance
    outputs = self.optimizer_loop.run(split_batch, optimizers, batch_idx)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/loops/base.py", line 145, in run
    self.advance(*args, **kwargs)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/loops/optimization/optimizer_loop.py", line 215, in advance
    result = self._run_optimization(
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/loops/optimization/optimizer_loop.py", line 266, in _run_optimization
    self._optimizer_step(optimizer, opt_idx, batch_idx, closure)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/loops/optimization/optimizer_loop.py", line 378, in _optimizer_step
    lightning_module.optimizer_step(
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/core/lightning.py", line 1652, in optimizer_step
    optimizer.step(closure=optimizer_closure)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/core/optimizer.py", line 164, in step
    trainer.accelerator.optimizer_step(self._optimizer, self._optimizer_idx, closure, **kwargs)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/accelerators/accelerator.py", line 336, in optimizer_step
    self.precision_plugin.optimizer_step(model, optimizer, opt_idx, closure, **kwargs)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/plugins/precision/precision_plugin.py", line 163, in optimizer_step
    optimizer.step(closure=closure, **kwargs)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/optim/optimizer.py", line 88, in wrapper
    return func(*args, **kwargs)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/autograd/grad_mode.py", line 28, in decorate_context
    return func(*args, **kwargs)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/optim/adam.py", line 92, in step
    loss = closure()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/plugins/precision/precision_plugin.py", line 148, in _wrap_closure
    closure_result = closure()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/loops/optimization/optimizer_loop.py", line 160, in __call__
    self._result = self.closure(*args, **kwargs)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/loops/optimization/optimizer_loop.py", line 142, in closure
    step_output = self._step_fn()
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/loops/optimization/optimizer_loop.py", line 435, in _training_step
    training_step_output = self.trainer.accelerator.training_step(step_kwargs)
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/accelerators/accelerator.py", line 216, in training_step
    return self.training_type_plugin.training_step(*step_kwargs.values())
  File "/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/plugins/training_type/training_type_plugin.py", line 213, in training_step
    return self.model.training_step(*args, **kwargs)
  File "/storage/scratch1/4/hmanjunatha6/carnet/src/models/encoding.py", line 171, in training_step
    output.reshape(self.batch_size * self.seq_length, *self.image_size),
RuntimeError: shape '[192, 1, 256, 256]' is invalid for input of size 10420224
Tue Oct 25 16:22:57 EDT 2022
Successfully completed
---------------------------------------
Begin PBS Epilogue Tue Oct 25 16:22:57 EDT 2022
Job ID:     5094433.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_segmentation
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=01:47:08,vmem=109833648kb,walltime=00:27:16,mem=3495196kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-005-28.pace.gatech.edu
End PBS Epilogue Tue Oct 25 16:22:57 EDT 2022
---------------------------------------
=>> PBS: job killed: walltime 30061 exceeded limit 30000
---------------------------------------
Begin PBS Epilogue Tue Oct 25 19:43:57 EDT 2022
Job ID:     5093390.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_segmentation
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=32:18:33,vmem=110014772kb,walltime=08:21:01,mem=3132488kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-01-004-31.pace.gatech.edu
End PBS Epilogue Tue Oct 25 19:43:57 EDT 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Tue Oct 25 21:30:18 EDT 2022
Job ID:     5095395.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn
Queue:      inferno
End PBS Prologue Tue Oct 25 21:30:18 EDT 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
[88mSkipping the block   |  carnet_autoencoder_training[0m
[88mSkipping the block   |  carnet_semseg_training[0m
[88mSkipping the block   |  verify_autoencoder[0m
[88mSkipping the block   |  verify_segmentation[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/nn/modules/lazy.py:178: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/utilities/model_summary.py:438: UserWarning: A layer with UninitializedParameter was found. Thus, the total number of parameters detected may be inaccurate.
  warning_cache.warn(

  | Name | Type   | Params
--------------------------------
0 | net  | CARNet | 395 K 
--------------------------------
395 K     Trainable params
0         Non-trainable params
395 K     Total params
1.583     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2022-10-25/CARNET exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  carnet_training[0m
Epoch    10: reducing learning rate of group 0 to 9.5000e-04.
=>> PBS: job killed: walltime 30053 exceeded limit 30000
---------------------------------------
Begin PBS Epilogue Wed Oct 26 05:51:09 EDT 2022
Job ID:     5095395.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=32:54:34,vmem=120506576kb,walltime=08:20:53,mem=3724728kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-005-30.pace.gatech.edu
End PBS Epilogue Wed Oct 26 05:51:09 EDT 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Wed Oct 26 14:19:24 EDT 2022
Job ID:     5098272.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn
Queue:      inferno
End PBS Prologue Wed Oct 26 14:19:24 EDT 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
[88mSkipping the block   |  carnet_autoencoder_training[0m
[88mSkipping the block   |  carnet_semseg_training[0m
[88mSkipping the block   |  verify_autoencoder[0m
[88mSkipping the block   |  verify_segmentation[0m
[88mSkipping the block   |  carnet_training[0m
[88mSkipping the block   |  verify_carnet[0m
[88mSkipping the block   |  rescarnet_training[0m
[88mSkipping the block   |  test_loss_function[0m
[88mSkipping the block   |  dataset_analysis[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/nn/modules/lazy.py:178: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
Missing logger folder: logs/2022-10-26/IMITATION/imitation_navigation
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/utilities/model_summary.py:438: UserWarning: A layer with UninitializedParameter was found. Thus, the total number of parameters detected may be inaccurate.
  warning_cache.warn(

  | Name | Type       | Params
------------------------------------
0 | net  | CIRLCARNet | 3.7 M 
------------------------------------
3.3 M     Trainable params
428 K     Non-trainable params
3.7 M     Total params
14.801    Total estimated model params size (MB)
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  imitation_with_carnet[0m
Epoch    19: reducing learning rate of group 0 to 9.5000e-04.
Epoch    26: reducing learning rate of group 0 to 9.0250e-04.
=>> PBS: job killed: walltime 60049 exceeded limit 60000
---------------------------------------
Begin PBS Epilogue Thu Oct 27 07:00:11 EDT 2022
Job ID:     5098272.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=16:40:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=65:43:59,vmem=109575572kb,walltime=16:40:49,mem=3714780kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-005-27.pace.gatech.edu
End PBS Epilogue Thu Oct 27 07:00:11 EDT 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Fri Oct 28 09:19:13 EDT 2022
Job ID:     5105154.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn
Queue:      inferno
End PBS Prologue Fri Oct 28 09:19:13 EDT 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
[88mSkipping the block   |  carnet_autoencoder_training[0m
[88mSkipping the block   |  carnet_semseg_training[0m
[88mSkipping the block   |  verify_autoencoder[0m
[88mSkipping the block   |  verify_segmentation[0m
[88mSkipping the block   |  carnet_training[0m
[88mSkipping the block   |  verify_carnet[0m
[88mSkipping the block   |  rescarnet_training[0m
[88mSkipping the block   |  test_loss_function[0m
[88mSkipping the block   |  dataset_analysis[0m
[88mSkipping the block   |  kalman_analysis[0m
[88mSkipping the block   |  imitation_with_carnet[0m
[88mSkipping the block   |  verify_carnet_imitation[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/nn/modules/lazy.py:178: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
Missing logger folder: logs/2022-10-28/IMITATION_KALMAN/imitation_navigation
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/utilities/model_summary.py:438: UserWarning: A layer with UninitializedParameter was found. Thus, the total number of parameters detected may be inaccurate.
  warning_cache.warn(

  | Name | Type       | Params
------------------------------------
0 | net  | CIRLCARNet | 3.7 M 
------------------------------------
3.3 M     Trainable params
428 K     Non-trainable params
3.7 M     Total params
14.801    Total estimated model params size (MB)
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  imitation_with_kalman_carnet[0m
---------------------------------------
Begin PBS Prologue Sat Nov  5 14:30:59 EDT 2022
Job ID:     5129138.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn
Queue:      inferno
End PBS Prologue Sat Nov  5 14:30:59 EDT 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
[88mSkipping the block   |  carnet_autoencoder_training[0m
[88mSkipping the block   |  carnet_semseg_training[0m
[88mSkipping the block   |  verify_autoencoder[0m
[88mSkipping the block   |  verify_segmentation[0m
[88mSkipping the block   |  carnet_training[0m
[88mSkipping the block   |  verify_carnet[0m
[88mSkipping the block   |  rescarnet_training[0m
[88mSkipping the block   |  test_loss_function[0m
[88mSkipping the block   |  dataset_analysis[0m
[88mSkipping the block   |  kalman_analysis[0m
[88mSkipping the block   |  imitation_with_carnet[0m
[88mSkipping the block   |  verify_carnet_imitation[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/nn/modules/lazy.py:178: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
Missing logger folder: logs/2022-11-05/IMITATION_KALMAN/imitation_navigation
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/utilities/model_summary.py:438: UserWarning: A layer with UninitializedParameter was found. Thus, the total number of parameters detected may be inaccurate.
  warning_cache.warn(

  | Name | Type       | Params
------------------------------------
0 | net  | CIRLCARNet | 3.7 M 
------------------------------------
3.3 M     Trainable params
428 K     Non-trainable params
3.7 M     Total params
14.801    Total estimated model params size (MB)
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  imitation_with_kalman_carnet[0m
=>> PBS: job killed: walltime 60060 exceeded limit 60000
---------------------------------------
Begin PBS Epilogue Sun Nov  6 06:11:59 EST 2022
Job ID:     5129138.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=16:40:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=66:11:59,vmem=109856504kb,walltime=16:41:00,mem=4030032kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-01-003-36.pace.gatech.edu
End PBS Epilogue Sun Nov  6 06:11:59 EST 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Wed Nov  9 11:36:23 EST 2022
Job ID:     5142948.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn
Queue:      inferno
End PBS Prologue Wed Nov  9 11:36:23 EST 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name | Type           | Params
----------------------------------------
0 | net  | CNNAutoEncoder | 308 K 
----------------------------------------
308 K     Trainable params
0         Non-trainable params
308 K     Total params
1.235     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2022-11-09/AUTOENCODER exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  carnet_autoencoder_training[0m
Epoch    21: reducing learning rate of group 0 to 9.5000e-04.
=>> PBS: job killed: walltime 30057 exceeded limit 30000
---------------------------------------
Begin PBS Epilogue Wed Nov  9 19:57:19 EST 2022
Job ID:     5142948.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=32:51:55,vmem=109418448kb,walltime=08:20:57,mem=3182448kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-003-34.pace.gatech.edu
End PBS Epilogue Wed Nov  9 19:57:19 EST 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Wed Nov  9 21:56:59 EST 2022
Job ID:     5144845.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn
Queue:      inferno
End PBS Prologue Wed Nov  9 21:56:59 EST 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name | Type           | Params
----------------------------------------
0 | net  | CNNAutoEncoder | 308 K 
----------------------------------------
308 K     Trainable params
0         Non-trainable params
308 K     Total params
1.235     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2022-11-09/AUTOENCODER exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  carnet_autoencoder_training[0m
Epoch    12: reducing learning rate of group 0 to 9.5000e-04.
Epoch    18: reducing learning rate of group 0 to 9.0250e-04.
Epoch    26: reducing learning rate of group 0 to 8.5737e-04.
=>> PBS: job killed: walltime 30054 exceeded limit 30000
---------------------------------------
Begin PBS Epilogue Thu Nov 10 06:18:03 EST 2022
Job ID:     5144845.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=32:55:28,vmem=112006016kb,walltime=08:21:04,mem=2940480kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-006-32.pace.gatech.edu
End PBS Epilogue Thu Nov 10 06:18:03 EST 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Mon Nov 14 01:41:44 EST 2022
Job ID:     5152686.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn
Queue:      inferno
End PBS Prologue Mon Nov 14 01:41:44 EST 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
[88mSkipping the block   |  carnet_autoencoder_training[0m
[88mSkipping the block   |  carnet_semseg_training[0m
[88mSkipping the block   |  verify_autoencoder[0m
[88mSkipping the block   |  verify_segmentation[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/nn/modules/lazy.py:178: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/utilities/model_summary.py:438: UserWarning: A layer with UninitializedParameter was found. Thus, the total number of parameters detected may be inaccurate.
  warning_cache.warn(

  | Name | Type   | Params
--------------------------------
0 | net  | CARNet | 407 K 
--------------------------------
407 K     Trainable params
0         Non-trainable params
407 K     Total params
1.631     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2022-11-14/CARNET exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  carnet_training[0m
Epoch    11: reducing learning rate of group 0 to 9.5000e-04.
=>> PBS: job killed: walltime 30047 exceeded limit 30000
---------------------------------------
Begin PBS Epilogue Mon Nov 14 10:02:31 EST 2022
Job ID:     5152686.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=08:20:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=32:55:13,vmem=125466152kb,walltime=08:20:47,mem=3708632kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-003-32.pace.gatech.edu
End PBS Epilogue Mon Nov 14 10:02:31 EST 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Mon Nov 14 10:13:26 EST 2022
Job ID:     5152940.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn
Queue:      inferno
End PBS Prologue Mon Nov 14 10:13:26 EST 2022
---------------------------------------
stty: standard input: Inappropriate ioctl for device
[88mSkipping the block   |  carnet_autoencoder_training[0m
[88mSkipping the block   |  carnet_semseg_training[0m
[88mSkipping the block   |  verify_autoencoder[0m
[88mSkipping the block   |  verify_segmentation[0m
[88mSkipping the block   |  carnet_training[0m
[88mSkipping the block   |  verify_carnet[0m
[88mSkipping the block   |  rescarnet_training[0m
[88mSkipping the block   |  test_loss_function[0m
[88mSkipping the block   |  dataset_analysis[0m
[88mSkipping the block   |  kalman_analysis[0m
[88mSkipping the block   |  imitation_with_carnet[0m
[88mSkipping the block   |  verify_carnet_imitation[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/nn/modules/lazy.py:178: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
Missing logger folder: logs/2022-11-14/IMITATION_KALMAN/imitation_navigation
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/utilities/model_summary.py:438: UserWarning: A layer with UninitializedParameter was found. Thus, the total number of parameters detected may be inaccurate.
  warning_cache.warn(

  | Name | Type       | Params
------------------------------------
0 | net  | CIRLCARNet | 1.6 M 
------------------------------------
1.2 M     Trainable params
440 K     Non-trainable params
1.6 M     Total params
6.444     Total estimated model params size (MB)
pygame 2.1.2 (SDL 2.0.16, Python 3.9.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
[1;32mRunning the block    |  imitation_with_kalman_carnet[0m
Epoch    18: reducing learning rate of group 0 to 9.5000e-04.
Epoch    24: reducing learning rate of group 0 to 9.0250e-04.
=>> PBS: job killed: walltime 60014 exceeded limit 60000
---------------------------------------
Begin PBS Epilogue Tue Nov 15 02:53:40 EST 2022
Job ID:     5152940.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=16:40:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=66:07:51,vmem=110997820kb,walltime=16:40:14,mem=4064356kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-006-34.pace.gatech.edu
End PBS Epilogue Tue Nov 15 02:53:40 EST 2022
---------------------------------------
---------------------------------------
Begin PBS Prologue Tue Jan  3 06:52:13 EST 2023
Job ID:     5305419.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_1
Queue:      inferno
End PBS Prologue Tue Jan  3 06:52:13 EST 2023
---------------------------------------
stty: standard input: Inappropriate ioctl for device
[88mSkipping the block   |  carnet_autoencoder_training[0m
[88mSkipping the block   |  carnet_semseg_training[0m
[88mSkipping the block   |  verify_autoencoder[0m
[88mSkipping the block   |  verify_segmentation[0m
[88mSkipping the block   |  carnet_training[0m
[88mSkipping the block   |  verify_carnet[0m
[88mSkipping the block   |  rescarnet_training[0m
[88mSkipping the block   |  test_loss_function[0m
[88mSkipping the block   |  dataset_analysis[0m
[88mSkipping the block   |  kalman_analysis[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/nn/modules/lazy.py:178: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/utilities/model_summary.py:438: UserWarning: A layer with UninitializedParameter was found. Thus, the total number of parameters detected may be inaccurate.
  warning_cache.warn(

  | Name | Type           | Params
----------------------------------------
0 | net  | CARNetExtended | 407 K 
----------------------------------------
407 K     Trainable params
0         Non-trainable params
407 K     Total params
1.631     Total estimated model params size (MB)
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/callbacks/model_checkpoint.py:631: UserWarning: Checkpoint directory /storage/scratch1/4/hmanjunatha6/carnet/logs/2023-01-03/CARNET_KALMAN exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
=>> PBS: job killed: walltime 43219 exceeded limit 43200
---------------------------------------
Begin PBS Epilogue Tue Jan  3 18:52:32 EST 2023
Job ID:     5305419.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_1
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=12:00:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=45:04:21,vmem=16053680kb,walltime=12:00:19,mem=2822472kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-006-36.pace.gatech.edu
End PBS Epilogue Tue Jan  3 18:52:32 EST 2023
---------------------------------------
---------------------------------------
Begin PBS Prologue Wed Jan  4 02:30:37 EST 2023
Job ID:     5305931.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_1
Queue:      inferno
End PBS Prologue Wed Jan  4 02:30:37 EST 2023
---------------------------------------
stty: standard input: Inappropriate ioctl for device
[88mSkipping the block   |  carnet_autoencoder_training[0m
[88mSkipping the block   |  carnet_semseg_training[0m
[88mSkipping the block   |  verify_autoencoder[0m
[88mSkipping the block   |  verify_segmentation[0m
[88mSkipping the block   |  carnet_training[0m
[88mSkipping the block   |  verify_carnet[0m
[88mSkipping the block   |  rescarnet_training[0m
[88mSkipping the block   |  test_loss_function[0m
[88mSkipping the block   |  dataset_analysis[0m
[88mSkipping the block   |  kalman_analysis[0m
[88mSkipping the block   |  carnet_with_kalman_training[0m
[88mSkipping the block   |  imitation_with_carnet[0m
[88mSkipping the block   |  verify_carnet_imitation[0m
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/torch/nn/modules/lazy.py:178: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
Missing logger folder: logs/2023-01-04/IMITATION_KALMAN/imitation_navigation
/storage/home/hcoda1/4/hmanjunatha6/.conda/envs/carla/lib/python3.9/site-packages/pytorch_lightning/utilities/model_summary.py:438: UserWarning: A layer with UninitializedParameter was found. Thus, the total number of parameters detected may be inaccurate.
  warning_cache.warn(

  | Name | Type       | Params
------------------------------------
0 | net  | CIRLCARNet | 1.6 M 
------------------------------------
1.2 M     Trainable params
441 K     Non-trainable params
1.6 M     Total params
6.448     Total estimated model params size (MB)
=>> PBS: job killed: walltime 60040 exceeded limit 60000
---------------------------------------
Begin PBS Epilogue Wed Jan  4 19:11:17 EST 2023
Job ID:     5305931.sched-torque.pace.gatech.edu
User ID:    hmanjunatha6
Job name:   rnn_1
Resources:  nodes=1:ppn=4:gpus=1:TeslaV100-16GB,pmem=8gb,walltime=16:40:00,neednodes=1:ppn=4:gpus=1:TeslaV100-16GB
Rsrc Used:  cput=66:36:57,vmem=12837996kb,walltime=16:40:40,mem=2659028kb,energy_used=0
Queue:      inferno
Nodes:     
atl1-1-02-003-34.pace.gatech.edu
End PBS Epilogue Wed Jan  4 19:11:17 EST 2023
---------------------------------------
